tracing-opentelemetry/src/lib.rs

```diff
@@ -144,3 +144,18 @@ pub struct OtelData {
     /// The otel span data recorded during the current tracing span.
     pub builder: opentelemetry::trace::SpanBuilder,
 }
+
+
+use std::time::SystemTime;
+
+#[doc(hidden)]
+#[cfg(not(target_arch = "wasm32"))]
+fn now() -> SystemTime {
+    SystemTime::now()
+}
+
+#[doc(hidden)]
```

Comment:
since these are private types, they shouldn't need to be `doc(hidden)`?
```suggestion
#[cfg(not(target_arch = "wasm32"))]
fn now() -> SystemTime {
    SystemTime::now()
}

```